diff --git a/lua/settings.lua b/lua/settings.lua
index 050b7a7..12447b1 100644
--- a/lua/settings.lua
+++ b/lua/settings.lua
@@ -37,6 +37,11 @@ vim.o.clipboard = "unnamedplus" -- Copy paste between vim and everything else
 -- vim.o.guifont = "JetBrainsMono\\ Nerd\\ Font\\ Mono:h18"
 -- vim.o.guifont = "Hack\\ Nerd\\ Font\\ Mono"
 -- vim.o.guifont = "SauceCodePro Nerd Font:h17"
-vim.o.guifont = "FiraCode Nerd Font:h17"
+vim.o.guifont = "CascadiaCode\\ NF:h13"
+
+-- Neovide Configuration
+vim.api.nvim_set_var('neovide_refresh_rate', 144)
+vim.api.nvim_set_var('neovide_transparency', 0.9)
+vim.api.nvim_set_var('neovide_cursor_vfx_mode', 'wireframe') -- railgun, torpedo, pixiedust, sonicboom, ripple, wireframe
 
 -- vim.o.guifont = "JetBrains\\ Mono\\ Regular\\ Nerd\\ Font\\ Complete"
