diff --git a/lua/lv-globals.lua b/lua/lv-globals.lua
index dedf397..0952f0f 100644
--- a/lua/lv-globals.lua
+++ b/lua/lv-globals.lua
@@ -2,11 +2,11 @@ O = {
     auto_close_tree = 0,
     auto_complete = true,
     colorscheme = 'lunar',
-    hidden_files = true,
+    hidden_files = false,
     wrap_lines = false,
     number = true,
     relative_number = true,
-    shell = 'bash',
+    shell = 'fish',
 
     -- @usage pass a table with your desired languages
     treesitter = {
@@ -14,7 +14,7 @@ O = {
         ignore_install = {"haskell"},
         highlight = {enabled = true},
         playground = {enabled = true},
-        rainbow = {enabled = false}
+        rainbow = {enabled = true}
     },
 
     database = {save_location = '~/.config/nvcode_db', auto_execute = 1},
